apps/transloco-playground/src/assets/i18n/custom-transpilers/es.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
I don't see the point in adding this to the docs, this is a very specific implementation you needed.

Reply:
I think you can make a simpler example by providing a custom transpiler that logs the paramaters.